aws/rust-runtime/aws-config/src/imds/client.rs

```diff
@@ -1011,7 +1011,6 @@ pub(crate) mod test {
     }
 
     /// Verify that the end-to-end real client has a 1-second connect timeout
-    #[ignore]
```

Comment:
Intentional?

Reply:
yeah figured that taking 1 second to actually make sure that timeouts really work was worth it